src/main/java/io/github/cottonmc/epicurean/item/EpicureanItems.java

```diff
@@ -48,6 +48,7 @@
 	public static final Item CHICKEN_TENDER = register("chicken_tender", new MealItem(5, 0.3f));
 	public static final Item HAMBURGER = register("hamburger", new MealItem(10, 0.5f));
 	public static final Item PANCAKE = register("pancake", new MealItem(5, 0.3f));
+	public static final Item SMOKED_SALMON = register("smoked_salmon", new MealItem(8, .6f));
```

Comment:
Don't want this to be a meal item. Just make it a normal Item with a food setting. Meals are designed to be made in the cooking table.

Reply:
Fixed